R/Figures.R

```diff
@@ -1,15 +1,17 @@
 #' Create Boxplot
 #' 
 #' @param filtered.data TADA data frame containing the data downloaded from the WQP, where
-#' each row represents a unique data record. Data frame must include the columns
-#' 'TADA.ComparableDataIdentifier", 'TADA.ResultMeasureValue', and
-#' 'TADA.ResultMeasure.MeasureUnitCode' to run this function. These columns can 
+#' each row represents a unique data record. Data frame must include the columns 'TADA.CharacteristicName' OR
+#' 'TADA.ComparableDataIdentifier', 'TADA.ResultMeasureValue', and
+#' 'TADA.ResultMeasure.MeasureUnitCode' to run this function. 'TADA.ComparableDataIdentifier' can 
 #' be added to the data frame by running the function HarmonizeData(). The data frame
-#' must be filtered down to a single comparable data identifier to run this function.
+#' must be filtered down to a single characteristic with a consistent unit or comparable data identifier to run this function.
+#' 
+#' @param id_col The column in the dataset used to identify the characteristic plotted in the boxplot. May be set to "TADA.CharacteristicName" or "TADA.ComparableDataIdentifier" if user would rather use a post-harmonization dataset that groups multiple TADA.CharacteristicNames into one. 
```

Comment:
In addition to grouping synonyms, TADA.ComparableDataIdentifier identifies comparable data groupings. This is the important piece to consider for plotting and generating stats. Specifically, this groups data with unique combinations of the characteristic name, speciation, and fraction. Without speciation and fraction included, the characteristic name alone may not be sufficient to determine the group of data (grouped by char alone) is comparable and should be plotted together. Nutrients are a good example.